api/src/main/java/org/pmiops/workbench/db/dao/UserServiceImpl.java

```diff
@@ -1174,6 +1179,19 @@ public void maybeSendAccessExpirationEmail(DbUser user) {
     }
   }
 
+  private boolean moduleIsEnabledInEnvironment(ModuleNameEnum moduleName) {
```

Comment:
we may be able to do something more elegant once more of https://precisionmedicineinitiative.atlassian.net/browse/RW-6237 is implemented